src/main/java/ogc/rs/apiserver/ApiServerVerticle.java

```diff
@@ -1206,7 +1202,150 @@ public void getStacCollection(RoutingContext routingContext) {
+                    // TODO: next link implementation
+//                    .add(new JsonObject()
+//                        .put("rel", "next")
+//                        .put("type", "application/geo+json")
+//                        .put("method", "GET")
+//                        .put("href", stacMetaJson.getString("hostname")
+//                            + "/stac/collections/" + collectionId + "/items?")));
+              } catch (Exception e) {
+                LOGGER.error("Something went wrong here: {}", e.getMessage());
+                routingContext.put(
+                    "response",
+                    new OgcException(500, "Internal Server Error", "Something broke")
+                        .getJson()
+                        .toString());
+                routingContext.put("statusCode", 500);
+                routingContext.next();
+              }
+              routingContext.put("response", featureCollections.toString());
+              routingContext.put("statusCode", 200);
+              routingContext.next();
+            })
+        .onFailure(
+            failed -> {
+              if (failed instanceof OgcException) {
+                routingContext.put("response", ((OgcException) failed).getJson().toString());
+                routingContext.put("statusCode", ((OgcException) failed).getStatusCode());
+              } else {
+                OgcException ogcException =
+                    new OgcException(500, "Internal Server Error", "Internal Server Error");
+                routingContext.put("response", ogcException.getJson().toString());
+                routingContext.put("statusCode", ogcException.getStatusCode());
+              }
+              routingContext.next();
+            });
+  }
+
+  // /item/<item-id> for stac_item
+  public void getStacItemById(RoutingContext routingContext){
+//    String stacCollectionId = routingContext.normalizedPath().split("/")[3];
+//    String stacItemId = routingContext.normalizedPath().split("/")[5];
+    String stacCollectionId = routingContext.pathParam("collectionId");
+    String stacItemId = routingContext.pathParam("itemId");
+        LOGGER.debug("stacCollectionId- {}, stacItemId- {}", stacCollectionId, stacItemId);
+    JsonArray commonLinksInFeature = new JsonArray()
+        .add(new JsonObject()
+            .put("rel", "collection")
+            .put("type", "application/json")
+            .put("href", stacMetaJson.getString("hostname") + "/stac/collections/" + stacCollectionId))
+        .add(new JsonObject()
+            .put("rel", "parent")
+            .put("type", "application/json")
+            .put("href", stacMetaJson.getString("hostname") + "/stac/collections/" + stacCollectionId))
+        .add(new JsonObject()
+            .put("rel", "root")
+            .put("type", "application/json")
+            .put("href", stacMetaJson.getString("hostname") + "/stac"));
+    dbService
+        .getStacItemById(stacCollectionId, stacItemId)
```

Comment:
Need to modify asset links to create `hostname/assets/<asset-id>` URL like in `stacCollections`